hack/README.md

```diff
@@ -0,0 +1,61 @@
+## Bringup KIND Cluster with multiple nodes
```

Comment:
Can you add a section with the purpouse of this readme?
I know it's about manually testing, but it's not clear.
Also, I believe it's better to have e2e automated tests than how to test manually.

In general, it would make sense to add a "contribution guidelines" with all the informations needed for developers. I'll file a task for that

Reply:
indeed e2e is the way to go this is my 1st code drop of the operator and I need a way to test it in standalone 
Now I know it more or less work I will remove them and just keep them local in my view